keyboards/sanctified/dystopia/readme.md

```diff
@@ -0,0 +1,15 @@
+# Dystopia
```

Comment:
Could you add instructions on how to enter the bootloader, to reflash the keyboard?

Reply:
Done in commit 05cf339